src/docs/metrics/contract-transactions-count/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Transactions Count
+author: Yury
+date: 2022-04-21
+description: Get transactions count for a chain address / contract
+---
+
+## Definition
+
+Get transactions count for a chain address / contract
+
+[Example of usage **transactions count** on Sanbase](https://app.santiment.net/s/rk-8It-D)
+
+---
+
+## Access
+
+[Free Access](/metrics/details/access#free-access)
+
+---
+
+## API
+
+Available under the `Transactions Count` name:
```

Comment:
The name in the API is `transactions_count` (no spaces, underscore, all lowercase)